components/contributors/ActivityCalendarGitHub.tsx

```diff
@@ -70,50 +70,32 @@ export default function ActivityCalendarGit({
   const yearDiff = Number(new Date().getFullYear()) - getFirstContribYear();
   const yearsList = lastNYears(yearDiff);
 
-  const [year, setYear] = useState(0);
+  const [year, setYear] = useState(new Date().getFullYear());
```

Comment:
The reason why we kept this as 0 is, if no year is selected, we would show past 365 days data. i.e. if we are on Jan 10th, 2025, we'll show Jan 10th 2024 to Jan 10th 2025's data and not just Jan 1st 2025 to Jan 10th 2025.